utils.go

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright (c) 2017 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package virtcontainers
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+)
+
+func fileCopy(srcPath, dstPath string) error {
```

Comment:
@jodh-intel I honestly don't know what is the best here. If @sameo agrees with this executing `cp` directly from our go code, then I am fine with this.
Let's just try to get this PR merged by end of this week at least, because it virtcontainers which is currently broken.